Makefile.am

```diff
@@ -11,15 +11,19 @@ else
 LIBVIRT_BUILD_TAG=
 endif
 
+COMMIT=`git describe --dirty --always --tags 2> /dev/null || true`
+VERSION=`cat ./VERSION`
```

Comment:
@WeiZhang555 I guess you don't need VERSION file, var VERSION already exists, https://github.com/hyperhq/runv/blob/master/configure.ac#L108